### ISSUE TITLE
Add option for server-wide default repository visibility

### ISSUE THREAD

**Issue by lafriks** [woodpecker-ci/woodpecker#1220]
### Clear and concise description of the problem

Currently when activating new repository it will always default to public.

### Suggested solution

This could be either global option in woodpecker server or it could use more sane defaults combining if Gitea server is private but repo/org visibility is public than it should still default to private etc

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by anbraten** [comment]
Normally they default to the repo visibility. So in case your repo is private in Github / Gitea ... its private in Woodpecker as well.

**Comment by lafriks** [comment]
Yes but if you have private gitea instance and public repo (still accessible only by authorization) you will have public woodpecker repository that will be accessible without authorization that is wrong imho
